region/maridia/outer.json

```diff
@@ -2454,6 +2550,45 @@
                 }
               ]
             },
+            {
+              "id": 5,
+              "strats": [
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    "Gravity"
+                  ],
+                  "note": "Kill or dodge the fish by the top left door."
+                }
+              ]
+            },
+            {
+              "id": 7,
+              "strats": [
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    "Gravity",
+                    "h_canArtificialMorphBombHorizontally"
```

Comment:
You've added the tech `canBombHorizontally` in some places and the helper in others, all next to `h_canArtificialMorphIBJ`.

Reply:
I'll standardize them. The helper is going to be a bit slower to compute, and adds ammo requirements (which wont matter since theyre with IBJs)